k8s.gcr.io/Vanity-Domain-Flip.md

```diff
@@ -0,0 +1,191 @@
+# The Problem
+
+Currently, `k8s.gcr.io` is a vanity domain that points to
+`gcr.io/google-containers` (Google-owned and managed). This is a problem because
+`k8s.gcr.io` is an alias used throughut the Kubernetes codebase. As Kubernetes
+is a community-owned project, `k8s.gcr.io` should instead point to a
+community-controlled repo.
+
+# The Solution
+
+The community has created a new repo called `gcr.io/k8s-artifacts-prod`, and it
+has been agreed that the community should use it as the new place to push
+production images (instead of `gcr.io/google-containers`). **We can solve the
+above problem by flipping the vanity domain (`k8s.gcr.io`) from
+`gcr.io/google-containers` to `gcr.io/k8s-artifacts-prod`**. This way, no change
+needs to be made in the Kubernetes codebase.
+
+The minimum prerequisite is that the existing images in `google-containers` must
+be copied into `k8s-artifacts-prod` in order to ensure that the domain flip
+happens transparently without incurring any interruptions. However there are
+other infrastructural improvements that the community has designed, such as
+explicit backups, disaster recovery, and also auditing and alerting.
+
+The rest of this document explains the infrastructural improvements surrounding
+`gcr.io/k8s-artifacts-prod`.
+
+# How New Images Get Promoted (Pushed) to Production
+
+To get new images into `gcr.io/google-containers`, a Googler must approve a
+change in Google's private repository.
+
+On the other hand, `gcr.io/k8s-artifacts-prod` is integrated with a
+publicly-visible GitHub repository, named [k8s.io][k8sio]. The [promoter][CIP]
+watches this repository for changes and promotes images. In addition, a system
+of setting up staging repos, and promoting from them into
+`gcr.io/k8s-artifacts-prod` has been [created][staging-subproject] so that owners of
```

Comment:
can you spell out `{us,eu,asia}.gcr.io` when referencing the new GCR.  The non-region-named version shows nothing if I go to that URL

Reply:
Ack